doc/man1/cms.pod

```diff
@@ -553,6 +558,36 @@ The B<-debug_decrypt> option can be used to disable the MMA attack protection
 and return an error if no recipient can be found: this option should be used
 with caution. For a fuller description see L<CMS_decrypt(3)>).
 
+=head1 CAdES Basic Electronic Signature (CAdES-BES)
+
+A CAdES Basic Electronic Signature (CAdES-BES), as defined in the European Standard ETSI EN 319 122-1 V1.1.1, contains:
+
+=over 4
+
+=item *
+
+The signed user data as defined in CMS (RFC 3852);
+
+=item *
+
+Content-type of the EncapsulatedContentInfo value being signed;
+
+=item *
+
+Message-digest of the eContent OCTET STRING within encapContentInfo being signed;
+
+=item *
+
+An ESS signing-certificate or ESS signing-certificate-v2 attribute, as defined in Enhanced Security Services (ESS), RFC 2634 and RFC 5035.
+An ESS signing-certificate attribute only allows for the use of SHA-1 as a digest algorithm.
+An ESS signing-certificate-v2 attribute allows for the use of any digest algorithm.
+
+=item *
+
+The digital signature value computed on the user data and, when present, on the signed attributes.
+
+=back
+
```

Comment:
> This feature might be added in a future version, however.

Of course, you should add this sentence only if you intend to implement the feature ;-)

Reply:
> Up to now, the CAdES support is only implemented for signing but not for signature verification. I'm not saying that this should be implemented here. On the contrary, it would be a subject for a separate pull request (if it will be implemented at all). But it might confuse the casual user that `openssl cms -sign -cades ...` creates the signing certificate attribute, but `openssl cms -verify -cades ...` ignores it. So there should at least be some documentation in the manual pages.
> 
> (@FdaSilvaYY I'm not sure whether the `openssl cms` app should issue a warning or not in the case when the `-cades` option is ignored. What do you think?)

A CAdES signature is basically a CMS Signature with a few more attributes, so no need to add warning.
But the print code of `openssl cms...` tool should be changed (later) to display CAdES signature details clearly !
Verification code for CAdES is already present in `ts_verify.c`, 
I can prepare a PR shortly if someone ask.